plugins/UM3NetworkPrinting/src/Network/SendMaterialJob.py

```diff
@@ -67,10 +68,10 @@ def _determineMaterialsToSend(local_materials: Dict[str, LocalMaterial],
     #   \param materials_to_send A set with id's of materials that must be sent.
     def _sendMaterials(self, materials_to_send: Set[str]) -> None:
         container_registry = CuraApplication.getInstance().getContainerRegistry()
-        material_manager = CuraApplication.getInstance().getMaterialManager()
-        material_group_dict = material_manager.getAllMaterialGroups()
+        all_materials = container_registry.findInstanceContainersMetadata(type = "material")
+        all_root_materials = {material["base_file"] for material in all_materials if "base_file" in material}  # Filters out uniques by making it a set. Don't include files without base file (i.e. empty material).
```

Comment:
I'd name this `all_root_material_ids` to distinguish it from material metadata/containers.

Reply:
Done: a3e9316f4858581863cd7ead9b2d80c6f30e308f